### ISSUE TITLE
Comments not appearing after post; returned values in json

### ISSUE THREAD

**Issue by schmidtl4** [Viima/jquery-comments#85]
Using PHP PDO and a sqlite database as a backend for comments, I've set up all of the fields for jquery-comments in accordance with the specified types (INTEGER, BOOLEAN, VARCHAR).  I've also created a fieldMappings: dictionary to translate back and forth. 
When I post a comment it records in the database just fine.  The recorded values (along with the newly created Integer id are returned as follows (the keys are the database keys which will be translated by the dictionary fieldMappings). Here's a sample returned JSON string:
{"parent_ID":null,
"created":"2017-01-16",
"modified":"2017-01-16",
"content":"test 4",
"fullname":"You",
"profilePictureURL":"\/_images\/user-icon.png",
"createdByCurrentUser":true,
"upvoteCount":0,
"userHasUpvoted":false,
"ID":44}

Upon returning this json string the new comment is displayed like this:
![image](https://cloud.githubusercontent.com/assets/1418698/21995390/89af29c8-dbd9-11e6-942a-98b9a865fe6e.png)

In other words, the comment is acknowledged but the date and content are not there.

**Comment by Taxman972** [comment]
Hi @schmidtl4 ,
Try to change `createdByCurrentUser` to `created_by_current_user` #68 
second when usin jQuery for the Ajax call do you use `dataType: 'json'` param ?
